spec/reporters/muted-console-reporter.js

```diff
@@ -0,0 +1,58 @@
+const util = require("util");
```

Comment:
is there an existing reporter that would fit out needs?

Reply:
We'd need a reporter that does what the default reporter does already + muting the console.

Reply:
I'm asking if there is an existing reporter that redirects the console to a file(s). Not why we need to change the default reporter.

Reply:
What I meant is: whether we use an existing reporter or a custom one, we'd still have to invoke jasmine as a library, as this is the only to keep the default reporter and adding a new one (CLI ony allows to override the default reporter).

> I'm asking if there is an existing reporter that redirects the console to a file(s)
Not that I know of.